### PR TITLE
docs(cfn-guard): Add missing Input Parameters documentation & Update …

### PR DESCRIPTION
**Pull request by dannyvassallo**
…help messaging

*Issue #, if available:*

548

*Description of changes:*

Adds missing context for Input Parameter usage on the `validate` documentation and updates the help text for input parameters to make a clear distinction between parameter files and data files.

---
*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by dannyvassallo** [comment]
https://github.com/aws-cloudformation/cloudformation-guard/issues/548

File: README.md [hunk 1]
```diff
@@ -392,6 +392,81 @@ spec:
 
 The container `app` does not contain CPU limits specified, which fails the overall evaluation as shown in the screenshot.
 
+#### Using Input Parameters
+
+Guard allows you to use input parameters for dynamic data lookups during validation. This feature is particularly useful when you need to reference external data in your rules.
```
Comment (joshfried-aws) [inline README.md:397]: Do we want to specify how guard doesn't allow keys input params to have conflicting paths?

**Review by joshfried-aws — COMMENTED** [review]
Looks good left a minor question

File: README.md (hunk 1 above)
Comment (dannyvassallo) [reply README.md:397]: Yup - will add this!

**Review by joshfried-aws — APPROVED** [review]

**Review by pamkuang — APPROVED** [review]

**Pull request merged**